lib/ui/views/messenger/layouts/messenger_discussion_page.dart

```diff
@@ -329,21 +330,47 @@ class _MessageCreationFormFees extends ConsumerWidget {
   }
 }
 
-class _MessagesList extends ConsumerWidget {
+class _MessagesList extends ConsumerStatefulWidget {
```

Comment:
Why did you create a consumerStateful ? sorry i don't get it

Reply:
I tried plenty of solutions to make it work, that's something that stayed like that. I will change it.

Reply:
setState(() {
      hasMessages = discussion.valueOrNull?.lastMessage != null;
    });

I can check if there are messages into a discussion only in a async way, I need to make a setState to refresh the UI when I know that there are messages or not in the discussion.

Reply:
Unfortunately there is no way for us to customize the empty view in the listview https://github.com/EdsonBueno/infinite_scroll_pagination/blob/master/lib/src/widgets/helpers/default_status_indicators/no_items_found_indicator.dart